### PR TITLE
docs: edit step 2 of getting started start-routing.md

### PR DESCRIPTION
**Pull request by kapunahelewong**
Copy edit of body copy and headers.

Verified that the steps work. 11/19/2020

### CODE REVIEW

File: aio/content/start/start-routing.md [hunk 1]
```diff
@@ -1,78 +1,77 @@
 # Adding navigation
 
-At the end of [part 1](start "Get started with a basic Angular app"), the online store application has a basic product catalog.
-The app doesn't have any variable states or navigation.
-There is one URL, and that URL always displays the "My Store" page with a list of products and their descriptions.
+This guide builds upon the first step of the Getting Started tutorial, [Get started with a basic Angular app](start "Get started with a basic Angular app").
```
Comment (aikidave) [inline aio/content/start/start-routing.md:3]:
```suggestion
This guide builds on the first step of the Getting Started tutorial, [Get started with a basic Angular app](start "Get started with a basic Angular app").
```

Minor nit: `upon` is a bit formal.

File: aio/content/start/start-routing.md [hunk 2]
```diff
@@ -1,78 +1,77 @@
...
 
-This guide shows you how to use Angular [routing](guide/glossary#router "Router definition") to give the user in-app navigation. In a single-page app, instead of loading new pages, you show different components and data to the user based on where the user is in the application.
+At this stage of development, the online store application has a basic product catalog.
 
-The router lets you display full product details in separate [views](guide/glossary#view "View definition"), each with its own URL. Routing enables navigation from one view to the next (within the same page) as users perform tasks such as the following:
+This step of the tutorial guides you in configuring your application so you can do the following:
```
Comment (aikidave) [inline aio/content/start/start-routing.md:7]:
```suggestion
In the following sections, you'll add the following features to the application:
```
Again, minor nit, but this puts the focus more on the user and tightens the language.

File: aio/content/start/start-routing.md [hunk 3]
```diff
@@ -1,78 +1,77 @@
...
 
-* Entering a URL in the address bar to navigate to a corresponding view.
-* Clicking links on the page to navigate to a new view.
-* Clicking the browser's back and forward buttons to navigate backward and forward through the browser history.
+* Enter a URL in the address bar to navigate to a corresponding product page.
```
Comment (aikidave) [inline aio/content/start/start-routing.md:9]:
```suggestion
* Type a URL in the address bar to navigate to a corresponding product page.
```

File: aio/content/start/start-routing.md [hunk 4]
```diff
@@ -1,78 +1,77 @@
...
+* Click the browser's back and forward buttons to navigate the browser history intuitively.
 
-## Registering a route
+{@a define-routes}
 
-The app is already set up to use the Angular `Router` and to use routing to navigate to the product list component you modified earlier. This section shows you how to define a route to show individual product details.
+## Associate a URL path with a component
 
-1. Generate a new component for product details. Give the component the name `product-details`.
+The application is already set up to use the Angular `Router` to navigate to the `ProductListComponent`.
```
Comment (aikidave) [inline aio/content/start/start-routing.md:17]:
```suggestion
The application already uses the Angular `Router` to navigate to the `ProductListComponent`.
```

File: aio/content/start/start-routing.md [hunk 5]
```diff
@@ -1,78 +1,77 @@
...
+This section shows you how to define a route to show individual product details.
 
-    Reminder: In the file list, right-click the `app` folder, choose `Angular Generator` and `Component`.
+1. Generate a new component for product details.
+    In the file list, right-click the `app` folder, choose `Angular Generator` and `Component`.
+    Give the component the name `product-details`.
```
Comment (aikidave) [inline aio/content/start/start-routing.md:22]:
```suggestion
    Name the component `product-details`.
```
Moving from a general verb (`give`) to a more specific one.

File: aio/content/start/start-routing.md [hunk 6]
```diff
@@ -1,78 +1,77 @@
...
 
 1. In `app.module.ts`, add a route for product details, with a `path` of `products/:productId` and `ProductDetailsComponent` for the `component`.
 
     <code-example header="src/app/app.module.ts" path="getting-started/src/app/app.module.ts" region="product-details-route">
     </code-example>
 
-    A route associates one or more URL paths with a component.
+1. Open `product-list.component.html` so you can update the `*ngFor` directive.
+    To iterate over the items in the  `products` array, add `index as productId` to the `*ngFor`.
+    Adding `index as productId`  assigns each index in the `products` array to the `productId` variable when iterating over the list.
```
Comment (aikidave) [inline aio/content/start/start-routing.md:31]: This section seems like it has multiple steps:

1. Open `product-list.component.html`.
2. Update the `*ngFor` directive to read as follows.
    <code snippet>
    This statement instructs Angular to iterate over the items in the `products` array and assigns each index in the array to the `productId` variable when iterating over the list.

File: aio/content/start/start-routing.md [hunk 7]
```diff
@@ -1,78 +1,77 @@
...
 
-1. The directive configures the component template to define how the user navigates to the route or URL. When the user clicks a product name, the app  displays the details for that product.
-
-    1. Open `product-list.component.html`.
-
-    1. Update the `*ngFor` directive to assign each index in the `products` array to the `productId` variable when iterating over the list.
-
-    1. Modify the product name anchor to include a `routerLink`.
+    Next, modify the product name anchor to include a `routerLink`.
```
Comment (aikidave) [inline aio/content/start/start-routing.md:33]: Shouldn't this be a numbered step?
```suggestion
    1. Modify the product name anchor to include a `routerLink` directive.
```

File: aio/content/start/start-routing.md [hunk 8]
```diff
@@ -1,78 +1,77 @@
...
     </code-example>
 
-      The RouterLink directive gives the router control over the anchor element. In this case, the route, or URL, contains one fixed segment, `/products`, while the final segment is variable, inserting the id property of the current product. For example, the URL for a product with an `id` of 1 will be similar to `https://getting-started-myfork.stackblitz.io/products/1`.
+    The `RouterLink` directive helps you customize the anchor element.
+    In this case, the route, or URL, contains one fixed segment, `/products`.
+    The final segment is variable, inserting the `id` property of the current product.
+    For example, the URL for a product with an `id` of 1 would be similar to `https://getting-started-myfork.stackblitz.io/products/1`.
 
-1. Test the router by clicking a product name. The app displays the product details component, which currently always says "product-details works!"
+1. Now when you click a product name, the application displays the `ProductDetailsComponent`, which currently says "product-details works!"
```
Comment (aikidave) [inline aio/content/start/start-routing.md:43]:
```suggestion
1. Verify that the router works as intended by clicking the product name. The application should display the `ProductDetailsComponent`, which currently says "product-details works!"
```

File: aio/content/start/start-routing.md [hunk 9]
```diff
@@ -1,78 +1,77 @@
...
 
+The `ProductDetailsComponent` handles the display of each product.
+The Angular Router displays components based on the browser's URL and [your defined routes](#define-routes).
 
-## Using route information
-
-The product details component handles the display of each product. The Angular Router displays components based on the browser's URL and your defined routes. This section shows you how to use the Angular Router to combine the `products` data and route information to display the specific details for each product.
-
-1. Open `product-details.component.ts`
+This section shows you how to use the Angular Router to combine the `products` data and route information to display the specific details for each product.
```
Comment (aikidave) [inline aio/content/start/start-routing.md:57]:
```suggestion
In this section, you'll use the Angular Router to combine the `products` data and route information to display the specific details for each product.
```

File: aio/content/start/start-routing.md [hunk 10]
```diff
@@ -1,78 +1,77 @@
...
-        </code-example>
-
-    1. Define the `product` property and inject the `ActivatedRoute` into the constructor by adding it as an argument within the constructor's parentheses.
+    <code-example header="src/app/product-details/product-details.component.ts" path="getting-started/src/app/product-details/product-details.component.1.ts" region="imports">
+    </code-example>
 
-        <code-example header="src/app/product-details/product-details.component.ts" path="getting-started/src/app/product-details/product-details.component.1.ts" region="props-methods">
-        </code-example>
+1. Define the `product` property.
+    Next, inject the `ActivatedRoute` into the `constructor()` by adding `private route: ActivatedRoute` as an argument within the constructor's parentheses.
```
Comment (aikidave) [inline aio/content/start/start-routing.md:65]:
```suggestion
1. Inject the `ActivatedRoute` into the `constructor()` by adding `private route: ActivatedRoute` as an argument within the constructor's parentheses.
```

This should be a step also, correct?

File: aio/content/start/start-routing.md [hunk 11]
```diff
@@ -1,78 +1,77 @@
...
-        The `ActivatedRoute` is specific to each routed component that the Angular Router loads. It contains information about the
-        route, its parameters, and additional data associated with the route.
+    <code-example header="src/app/product-details/product-details.component.ts" path="getting-started/src/app/product-details/product-details.component.1.ts" region="props-methods">
+    </code-example>
 
-        By injecting the `ActivatedRoute`, you are configuring the component to use a *service*. The [Managing Data](start/start-data "Try it: Managing Data") page covers services in more detail.
+    The `ActivatedRoute` is specific to each component that the Angular Router loads.
+    `ActivatedRoute` contains information about the route and the route's parameters.
 
+    By injecting the `ActivatedRoute`, you are configuring the component to use a service.
```
Comment (aikidave) [inline aio/content/start/start-routing.md:73]:
```suggestion
    By injecting the `ActivatedRoute`, you are configuring the component to use a service.
```
I keep tripping up over phrases like " the `ActivatedRoute`" or "the `*ngFor`." I think it reads better if you either drop the article or add what the item is: the `ActivatedRoute` interface, or the `*ngFor` directive.

I'm not sure though. Maybe it's just me? Any thoughts?

File: aio/content/start/start-routing.md [hunk 12]
```diff
@@ -81,39 +80,31 @@ The product details component handles the display of each product. The Angular R
...
+    The line, `<h4>{{ product.price | currency }}</h4>`, uses the `currency` pipe to transform `product.price` from a number to a currency string.
+    A pipe is a way you can transform data in your HTML template.
+    For more information about Angular pipes, see [Pipes](guide/pipes "Pipes").
 
-    The line, `<h4>{{ product.price | currency }}</h4>`, uses the `currency` pipe to transform `product.price` from a number to a currency string. A pipe is a way you can transform data in your HTML template. For more information about Angular pipes, see [Pipes](guide/pipes "Pipes").
-
-    </div>
-
-Now, when users click on a name in the product list, the router navigates them to the distinct URL for the product, swaps out the product list component for the product details component, and displays the product details.
+Now, when users click on a name in the product list, the router navigates them to the distinct URL for the product, shows the `ProductDetailsComponent`, and displays the product details.
```
Comment (aikidave) [inline aio/content/start/start-routing.md:93]:
```suggestion
When users click on a name in the product list, the router navigates them to the distinct URL for the product, shows the `ProductDetailsComponent`, and displays the product details.
```

Our docs like to say "Now..." a lot. I don't think it's necessary.

File: aio/content/start/start-routing.md [hunk 13]
```diff
@@ -81,39 +80,31 @@ The product details component handles the display of each product. The Angular R
...
-## Next steps
+<hr />
 
-Congratulations! You have integrated routing into your online store.
+## What's next
 
-* Products are linked from the product list view to individual products.
-* Users can click on a product name from the list to see details in a new view, with a distinct URL/route.
+Congratulations!
+You have configured your application so you can view product details, each with a distinct URL.
```
Comment (aikidave) [inline aio/content/start/start-routing.md:106]:
```suggestion
In this section of the Getting Started tutorial, you configured your application so you can view product details, each with a distinct URL.
```
I am always wary of saying congratulations when I cannot say for sure that the user was successful. :)

File: aio/content/start/start-routing.md [hunk 14]
```diff
@@ -81,39 +80,31 @@ The product details component handles the display of each product. The Angular R
...
 
-To continue exploring Angular, choose either of the following options:
-* [Continue to the "Managing Data" section](start/start-data "Try it: Managing Data") to add a shopping cart feature, use a service to manage the cart data and use HTTP to retrieve external data for shipping prices.
-* [Skip ahead to the Deployment section](start/start-deployment "Try it: Deployment") to deploy your app to Firebase or move to local development.
+To continue exploring Angular, you can choose one of the following options:
```
Comment (aikidave) [inline aio/content/start/start-routing.md:108]:
```suggestion
To continue exploring Angular:
```

File: aio/content/start/start-routing.md (hunk 11 above)
Comment (kapunahelewong) [reply aio/content/start/start-routing.md:73]: I'm not a huge fan of the definite article in front of these either and usually remove them, depending on context. I'm happy to remove them in general. Currently, the style is in about half of the docs, maybe a little less now.

**Review by aikithoughts — APPROVED** [review]
LGTM!

Reviewed-for: global-docs-approvers

**Pull request closed without merging**